cli/src/install.sh

```diff
@@ -7,156 +7,146 @@ NC='\033[0m'
-      rc_path=".zshrc"
+    rc_path="${HOME}/.zshrc"
 
-      if ! grep -q '.phylum/completions' "$HOME/${rc_path}"; then
-          mkdir -p "$HOME/.phylum/completions"
-          echo "fpath+=(\"$HOME/.phylum/completions\")" >> "${HOME}/${rc_path}"
-      fi
-      if ! grep -q 'autoload -U compinit && compinit' "$HOME/${rc_path}"; then
-          echo "autoload -U compinit && compinit" >> "${HOME}/${rc_path}"
-      fi
+    if [[ ! -f "${rc_path}" ]]; then
+        touch "${rc_path}"
+    fi
 
-      success "Enabled completions for zsh."
+    if ! grep -q '.phylum/completions' "${rc_path}"; then
+        echo "fpath+=(\"\$HOME/.phylum/completions\")" >> "${rc_path}"
+    fi
+    if ! grep -q 'autoload -U compinit && compinit' "${rc_path}"; then
+        echo "autoload -U compinit && compinit" >> "${rc_path}"
+    fi
+    success "Completions are enabled for zsh."
 
-      add_to_path_and_alias "${rc_path}" "zsh"
-  fi
+    add_to_path_and_alias "${rc_path}" "zsh"
 }
 
 patch_bashrc() {
-  if [ -f "${HOME}/.bashrc" ]; then
-      rc_path="${HOME}/.bashrc"
+    rc_path="${HOME}/.bashrc"
 
-      if ! grep -q '.phylum/completions/phylum.bash' "${rc_path}"; then
-          echo "source \$HOME/.phylum/completions/phylum.bash" >> "${rc_path}"
-      fi
+    if [[ ! -f "${rc_path}" ]]; then
+        touch "${rc_path}"
+    fi
 
-      success "Enabled completions for bash."
+    if ! grep -q '.phylum/completions/phylum.bash' "${rc_path}"; then
+        echo "source \$HOME/.phylum/completions/phylum.bash" >> "${rc_path}"
+    fi
+    success "Completions are enabled for bash."
 
-      add_to_path_and_alias "${rc_path}" "bash"
-  fi
+    add_to_path_and_alias "${rc_path}" "bash"
 }
 
 create_directory() {
-  # Create the config directory if one does not already exist.
-  if [ ! -d "${HOME}/.phylum" ]; then
-      mkdir -p "${HOME}/.phylum"
-      success "Created directory .phylum in home directory."
-  fi
+    # Create the config directory if one does not already exist.
+    if [[ ! -d "${HOME}/.phylum" ]]; then
+        mkdir -p "${HOME}/.phylum"
+        success "Created directory .phylum in home directory."
+    fi
 }
 
 copy_files() {
-  # Copy the specific platform binary.
-  platform=$(get_platform)
-  arch=$(get_arch)
-  workdir=$(dirname "$0")
-  bin_name="phylum"
-
-  if [[ "$platform" == "macos" ]]; then
-      cat "${bin_name}" > "${HOME}/.phylum/phylum"
-      success "Copied ${bin_name} to ${HOME}/.phylum/phylum."
-  else
-      check_copy "${bin_name}" "${HOME}/.phylum/phylum"
-  fi
-  chmod +x "${HOME}/.phylum/phylum"
-
-  # Copy the settings over, if settings do not already exist at the target.
-  if [ ! -f "${HOME}/.phylum/settings.yaml" ]; then
-      check_copy "settings.yaml" "${HOME}/.phylum/"
-  fi
-
-  # Copy completions over
-  mkdir -p "${HOME}/.phylum/completions"
-  cp -a "completions" "${HOME}/.phylum/"
-  success "Copied completions."
+    # Copy the specific platform binary.
+    platform=$(set -e; get_platform)
```

Comment:
These fixes are in response to the `shellcheck` finding [SC2311](https://github.com/koalaman/shellcheck/wiki/SC2311), which appears when using the `-o all` option that is available starting with v0.8.0+

> Bash implicitly disabled set -e for this function invocation because it's inside a command substitution. Add set -e; before it or enable inherit_errexit.

This could have also been addressed with a single entry at the top of the file:
`shopt -s inherit_errexit`

If/when #216 is completed, this script will go from a bash one to portable one and the use of `set -e` for individual function invocations will go away.